constraint_utils/generic.py

```diff
@@ -34,6 +34,14 @@ def flip_base(base: hl.expr.StringExpression) -> hl.expr.StringExpression:
 
 
 def collapse_strand(ht: Union[hl.Table, hl.MatrixTable]) -> Union[hl.Table, hl.MatrixTable]:
+    """Flips the base in 'ref', 'alt', 'context' columns if either 'G' or 'T' exists in 'ref' column.
```

Comment:
might be good to add a quick note that this function calls `reverse_complement_bases`, which has been made obsolete by https://hail.is/docs/0.2/functions/genetics.html#hail.expr.functions.reverse_complement